docs/v2/manual.md

```diff
@@ -1150,6 +1150,45 @@ The default version flag (`-v/--version`) is defined as `cli.VersionFlag`, which
 is checked by the cli internals in order to print the `App.Version` via
 `cli.VersionPrinter` and break execution.
 
+
+### Timestamp Flag
+
+Using the timestamp flag is simple, You can look at time.Parse to get layout examples : https://golang.org/pkg/time/#example_Parse 
+
+<!-- {
+  "args": ["&#45;&#45;meeting", "2019-08-12T15:04:05"],
```

Comment:
⭐⭐⭐ _(must change)_ It took me a few seconds to figure out what I was reading here, can you add an example invoke to these docs? Like this:

```
myapp --meeting "2019-08-12T15:04:05"
```